onedal/common/_spmd_policy.py

```diff
@@ -0,0 +1,29 @@
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+from onedal import _backend, _is_dpc_backend
+import sys
+
+if _is_dpc_backend:
+    class _SPMDDataParallelInteropPolicy(_backend.spmd_data_parallel_policy):
```

Comment:
Duplicate class

Reply:
It is the main entity. The class definition in the _policy.py file was deleted